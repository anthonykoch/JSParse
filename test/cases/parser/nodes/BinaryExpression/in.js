10 in 42
