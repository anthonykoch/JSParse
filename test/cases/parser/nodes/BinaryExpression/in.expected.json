---
name: BinaryExpression - in
description: parses simple in binary
---

[
  {
    "type": "ExpressionStatement",
    "start": 0,
    "end": 8,
    "expression": {
      "type": "BinaryExpression",
      "operator": "in",
      "start": 0,
      "end": 8,
      "left": {
        "type": "Literal",
        "value": "10",
        "start": 0,
        "end": 2
      },
      "right": {
        "type": "Literal",
        "value": "42",
        "start": 6,
        "end": 8
      }
    }
  }
]
